README.md

```diff
@@ -371,6 +380,15 @@ var objectElement = new minim.ObjectElement({ foo: 'bar' });
 var value = objectElement.get('foo') // returns string instance for 'bar'
 ```
 
+##### getValue
+
+The `getValue` method returns the value of the `ObjectElement` instance at the given name.
+
+```javascript
+var objectElement = new minim.ObjectElement({ foo: 'bar' });
+var value = objectElement.getValue('foo') // returns 'bar'
```

Comment:
seems just `objectElement.value(0)` would be sufficient. As a rubyist my pull is to keep `get` and `set` out of method names, so I am biased.